pkg/spec/postgresql.go

```diff
@@ -165,9 +170,13 @@ func (m *MaintenanceWindow) UnmarshalJSON(data []byte) error {
 		return fmt.Errorf("'From' time must be prior to the 'To' time")
```

Comment:
no, we do not allow such period. One should split it into two:
- 15:35-23:59
- 00:00-13:25